src/Adapter/Security/Admin.php

```diff
@@ -77,6 +77,13 @@ public function onKernelRequest(GetResponseEvent $event)
             return true;
         }
 
+        // in case of exception handler sub request, avoid infinite redirection
+        if ($event->getRequestType() === HttpKernelInterface::SUB_REQUEST
+            && isset($event->getRequest()->attributes['exception'])
```

Comment:
The symfony exception handler screen works as an internal sub request. We can identify that (a sub request with an exception attribute) and skip the authentification/redirection part.